### PR TITLE
cli53 0.7.4 (new formula)

### PR DESCRIPTION
**Pull request by blackjid**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
### Description

cli53 provides import and export from BIND format and simple command line management of Route 53 domains.

### CODE REVIEW

File: Formula/cli53.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    mkdir_p buildpath/"src/github.com/barnybug"
+    ln_s buildpath, buildpath/"src/github.com/barnybug/cli53"
+
+    system "make", "build"
+
+    bin.install "cli53"
+  end
+
+  test do
+    system "#{bin}/cli53", "--version"
```
Comment (DomT4) [inline Formula/cli53.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Formula/cli53.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p buildpath/"src/github.com/barnybug"
+    ln_s buildpath, buildpath/"src/github.com/barnybug/cli53"
+
+    system "make", "build"
+
```
Comment (DomT4) [inline Formula/cli53.rb]: Probably don't need this empty line.

File: Formula/cli53.rb (hunk 1 above)
Comment (blackjid) [inline Formula/cli53.rb]: all cli53 need a AWS id and secret to run. Also, all make a request to amazon and a HTTP mock maybe is too much.

Maybe check that running `cli53` without args prints something expected, like `cli53 - manage route53 DNS`

Any thoughts?

```
NAME:
   cli53 - manage route53 DNS

USAGE:
   cli53 [global options] command [command options] [arguments...]

COMMANDS:
   list, l	list domains
   create	create a domain
   delete	delete a domain
   import	import a bind zone file
   export	export a bind zone file (to stdout)
   rrcreate, rc	create one or more records
   rrdelete, rd	delete a record
   rrpurge	delete all the records (danger!)
   dslist	list reusable delegation sets
   dscreate	create a reusable delegation set
   dsdelete	delete a reusable delegation set
   help, h	Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --help, -h		show help
   --version, -v	print the version
```

Comment (DomT4) [inline Formula/cli53.rb]: Tweaked it slightly in [post](https://github.com/Homebrew/homebrew-core/commit/e1a7145e900a67a53de5f6f65a66e366cd6c451b#diff-9ca55149f206b01dbb5e693fbd58b57dR19) but yeah, there's not much we can do here on functionality given the credential issue. There's a few formulae like this, so it's not unheard of.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/e1a7145e900a67a53de5f6f65a66e366cd6c451b. Thank you for your contribution to Homebrew @blackjid; we appreciate it! :smiley_cat:

**Pull request closed without merging**